fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -834,6 +834,7 @@ public void transferToNonLeader(FrontendNodeType newType) {
 
     public static void destroyCheckpoint() {
         if (CHECKPOINT != null) {
+            CHECKPOINT.shutdown();
```

Comment:
Better to add a try catch, because this is the key path of the checkpoint.

Reply:
got it. fixed